js/huobi.js

```diff
@@ -2259,6 +2263,8 @@ module.exports = class huobi extends Exchange {
                 'code': code,
                 'id': currencyId,
                 'active': currencyActive,
+                'deposit': undefined,
+                'withdraw': undefined,
```

Comment:
We can set `withdraw` and `deposit` here based on the chains information. If there's just one chain, and it is withdrawable and depositable - the entire currency is also depositable and withdrawable. If there's more than one chain, and all chains are both withdrawable and depositable – the entire currency is also depositable and withdrawable as well.